service-config/src/main/java/com/palantir/config/service/CustomImmutableStyle.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.config.service;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.immutables.value.Value.Style;
+
+/**
+ * Custom style for immutable classes.
+ */
+@Target({ElementType.PACKAGE, ElementType.TYPE})
```

Comment:
removed @Retention, but kept @Target since we only want the style to be applied at the package or the class level